### PR TITLE
Added Basic UI

### PR DESCRIPTION
**Pull request by Lorenzo-Mariano**

### CODE REVIEW

File: packages/frontend/src/app/globals.css [hunk 1]
```diff
@@ -1,27 +1,21 @@
...
-
-:root {
-  --foreground-rgb: 0, 0, 0;
-  --background-start-rgb: 214, 219, 220;
-  --background-end-rgb: 255, 255, 255;
+/* *,
+* *,
+* * *,
+* * * * {
+  outline: 1px solid blueviolet;
```
Comment (mikolertesx) [inline packages/frontend/src/app/globals.css]: Could you remove this comments?

File: packages/frontend/src/app/layout.tsx [hunk 2]
```diff
@@ -1,21 +1,23 @@
-import './globals.css'
-import { Inter } from 'next/font/google'
+import "./globals.css";
+import "../../node_modules/primeflex/primeflex.css";
```
Comment (mikolertesx) [inline packages/frontend/src/app/layout.tsx]: Could you change the import to be "/node_modules/primeflex/primeflex.css", it doesn't have to be relative if it's not necessary

File: packages/frontend/src/app/page.tsx [hunk 3]
```diff
@@ -11,22 +11,146 @@ import "primereact/resources/primereact.min.css";
...
+
+	// perhaps friends list on the right and chat window
+	// on the right.
+	return (
+		// <main className="flex min-h-screen flex-col items-center justify-between p-24">
+		// 	<Button icon={"pi pi-check"} label="Start Chatting!"></Button>
+		// </main>
+
+		// soy mejicano!!! jajajajaja
+		//                   - monika
```
Comment (mikolertesx) [inline packages/frontend/src/app/page.tsx]: 😆 You caught me.

**Review by mikolertesx — CHANGES_REQUESTED** [review]
Some important changes

File: packages/frontend/src/app/globals.css (hunk 1 above)
Comment (Lorenzo-Mariano) [reply packages/frontend/src/app/globals.css]: Understood

File: packages/frontend/src/app/layout.tsx (hunk 2 above)
Comment (mikolertesx) [reply packages/frontend/src/app/layout.tsx]: Could you rewrite it to say 
```jsx
import "/node_modules/primeflex/primeflex.css";
```

**Review by mikolertesx — APPROVED** [review]
LGTM

**Pull request merged**